### ISSUE TITLE
fxLayoutWrap no longer exists

### ISSUE THREAD

**Issue by jihuiwang** [dschnelldavis/angular2-json-schema-form#204]
I was using material design framework but received an error
_Can't bind to 'fxLayoutWrap' since it isn't a known property of 'flex-layout-root-widget'._

I checked angular/flex-layout on github and found that fxLayoutWrap was removed since Nov 30, 2017.

**Comment by jihuiwang** [comment]
it is used in 'src/lib/src/framework-library/material-design-framework/flex-layout-section.component.ts'

**Comment by oliverrenner** [comment]
same issue here!!

**Comment by jihuiwang** [comment]
@oliverrenner 
remove the [fxLayoutWrap] Directive from the template in 'src/lib/src/framework-library/material-design-framework/flex-layout-section.component.ts'
make it work

**Comment by Koslun** [comment]
@jihuiwang
To be precise, it was removed in 5.0.0-beta.13 on 22nd February in 2018 but was deprecated earlier, maybe the date you mentioned. See https://github.com/angular/flex-layout/blob/master/CHANGELOG.md#breaking-changes

If the `fxLayoutWrap` directive is there a corresponding `wrap` should likely be added to the `fxLayout` property to keep the same functionality. I.e. from `fxLayout="row"` to  `fxLayout="row wrap"`

**Comment by beeing** [comment]
use `fxLayout="row wrap"`

**Comment by jihuiwang** [comment]
@Koslun 
yes, the error appears with the latest version of angular/flex-layout, which is 5.0.0-beta.13
now i roll back to the previous one (2.0.0-beta.12).
and it works again.

**Comment by Koslun** [comment]
@jihuiwang yes, I've done the same for now.

**Comment by rroeneman** [comment]
Work around, removing the (part of) line
[fxLayoutWrap]=\"options?.fxLayoutWrap\"\n
from angular2-json-schema-form.es5 file, worked for me to upgrade to flex-layout 5.0.0-beta.13.
Not fully tested yet, but could help...

**Comment by vatsalpande** [comment]
Any update on this issue? Facing the same

**Comment by Koslun** [comment]
@vatsalpande fixed in PR #230

**Comment by catchshyam** [comment]
Any ETA on the release that includes PR #230 ?

**Comment by Theyssen** [comment]
Does anyone have a solution in anticipation of the pull request?

**Comment by Koslun** [comment]
@Theyssen Using a forked version of this repo containing the above PR. Steps to do so are described in above PR: https://github.com/dschnelldavis/angular2-json-schema-form/pull/230#issuecomment-383591628
